dbfit-java/build.gradle

```diff
@@ -55,6 +55,25 @@ allprojects {
     dependencies {
         compile "org.fitnesse:fitnesse:${fitNesseVersion}:standalone@jar"
     }
+    
+    // fix gradle .classpath generation for bin folders and put it under /build/eclipse
+	eclipse {
```

Comment:
Formatting remark: Indentation is a bit inconsistent here. We're using 4-space indentation. No tabs. No trailing spaces on lines. Also better to surround `if` blocks with some empty lines to make it easier to see.

Reply:
good catch! I'll try to get a formatter for gradle files and I've forced space indentation instead of autodetection in this workspace.